bin.js

```diff
@@ -31,3 +34,12 @@ function getConfig() {
   }
 }
 
+function getRules(set) {
+  var configFileName = set ? ('eslint-config-' + set) : null
```

Comment:
Yeah, sorry, a little more context:

The purpose of this tool is to determine if you have explicitly configured all the available rules. Using `getConfigForFile` would probably be correct, but am I correct in assuming that it will also fill the config with the defaults from ESLint? If that's the case that wouldn't work for our use case.